### PR TITLE
Add script that allows executing a command with the docker compose env

### PR DESCRIPTION
**Pull request by frederikrothenberger**
The script encapsulates starting and stopping the docker environment and executes the supplied command while the environment is up.

### CODE REVIEW

File: scripts/with-docker-env [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+scripts/start-selenium-env
+
+ret=0
+"$@" || ret="$?"
+
+echo "'$*' returned with $ret"
+
+cd docker-test-env
+docker compose down
```
Comment (nmattia) [inline scripts/with-docker-env:42]: what happens if docker compose is not up? Could this already return with non-zero?

**Review by nmattia — APPROVED** [review]
nice! do we need both scripts? we could for instance have one script with two (very similar) behaviors:

``` bash
$ ./scripts/with-docker-env my-command # runs until my-command has returned, and then tear down env
$ ./scripts/with-docker-env # runs until users hits Ctrl+C, i.e. uses something like `sleep infinity` as the command
```

wdyt?

File: scripts/with-docker-env (hunk 1 above)
Comment (frederikrothenberger) [reply scripts/with-docker-env:42]: If `docker compose` fails, you'll get the error output from docker and a non-zero exit code. E.g.:
```
$ scripts/with-docker-env npm run test:e2e-desktop
[+] Building 0.0s (0/0)
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?: driver not connecting
```

Comment (frederikrothenberger) [reply scripts/with-docker-env:42]: Ah, and yes, failure of `docker compose down` will cause non-zero exit code.

Comment (nmattia) [reply scripts/with-docker-env:42]: meaning the script will exit and you won't know what `$ret` was, correct? is it worth also "catching" the return code of `docker compose down` and then show both return codes before exiting?

Comment (frederikrothenberger) [reply scripts/with-docker-env:42]: `echo "'$*' returned with $ret"` shows the return code?

Comment (frederikrothenberger) [reply scripts/with-docker-env]: Added an explicit warning when `docker compose` fails to shut down properly.

**Pull request merged**